openlibrary/core/admin.py

```diff
@@ -7,7 +7,7 @@
 from infogami import config
 from infogami.utils import stats
 
-from . import cache
+from openlibrary.core import cache
```

Comment:
After the fact, but this edit doesn't match the commit message. It fixes Py3 import behavior, not .next() behavior.